### PR TITLE
basics

### PR DESCRIPTION
**Pull request by doomsayer13**

### CODE REVIEW

File: content/themes/casper/index.hbs [hunk 1]
```diff
@@ -20,8 +18,66 @@
...
+                    </div>
+                </li>
+                <li class="top">
+                    <div class="icon twitter">
+                        <a href="https://twitter.com/bitrise">Twitter</a>
+                    </div>
+                </li>
+                <li class="top">
+                    <div class="icon github">
+                        <a href="https://github.com/bitrise-iobitrise">Github</a>
```
Comment (vasarhelyia) [inline content/themes/casper/index.hbs]: please correct the link to https://github.com/bitrise-io

File: content/themes/casper/index.hbs [hunk 2]
```diff
@@ -20,8 +18,66 @@
...
+                    </div>
+                </li>
+                <li class="top">
+                    <div class="icon bitrise">
+                        <a href="https://www.bitrise.io/">Bitrise.io</a>
+                    </div>
+                </li>
+                <li class="top last">
+                    <div class="icon updates">
+                        <a href="http://blog.bitrise.io/">Bitrise Updates</a>
```
Comment (vasarhelyia) [inline content/themes/casper/index.hbs]: make it `Bitrise News` pls

**Comment by vasarhelyia** [comment]
please remove the `content/data/ghost-dev.db` file from GitHub and add it to `.gitignore`. everyone who's testing should have an own development db locally, but it shouldn't be shared across the team.

File: content/themes/casper/assets/css/screen.css [hunk 3]
```diff
@@ -1742,28 +2026,71 @@ body:not(.post-template) .post-title {
...
    ========================================================================== */
 
-@media only screen and (max-width: 500px) {
+@media only screen and (max-width: 768px) {
 
-    .main-header {
-        margin-bottom: 15px;
-        height: 40vh;
+    .content {
+        flex-direction: column-reverse;
```
Comment (erosdome) [inline content/themes/casper/assets/css/screen.css:2059]: make it cross-browser compatible:
flex-direction, -moz-flex-direction, -webkit-flex-direction

File: content/themes/casper/assets/css/screen.css [hunk 4]
```diff
@@ -1742,28 +2026,71 @@ body:not(.post-template) .post-title {
...
+    }
+
+    .content .articles {
+        order: 2;
```
Comment (erosdome) [inline content/themes/casper/assets/css/screen.css:2067]: make it cross-browser compatible:
order, -moz-order, -webkit-order

File: content/themes/casper/assets/css/screen.css [hunk 5]
```diff
@@ -1742,28 +2026,71 @@ body:not(.post-template) .post-title {
...
+    }
+
+    .content .articles-addon {
+        max-width: 100%;
+        border-left: 0;
+        order: 1;
```
Comment (erosdome) [inline content/themes/casper/assets/css/screen.css:2077]: make it cross-browser compatible:
order, -moz-order, -webkit-order

**Pull request merged**